### ISSUE TITLE
Add more DVG4013 features

### ISSUE THREAD

**Issue by andig** [gonium/gosdm630#62]
The documentation seems to indicate that this meter supports more registers than currently implemented. It would be great to add those.

I would appreciate if anybody with a corresponding meter would get in contact.

**Comment by ionfan** [comment]
i have this meter and i am very interested in improvements regarding more support for the DVG4013.
Especially the current wattage would be intersting to view.
What information do you need? I attached a screenshot from the vendor-software so that you can see which values can be read.
![dvh4013](https://user-images.githubusercontent.com/47366151/52298247-df477400-2982-11e9-8101-ebc52705b647.jpg)

**Comment by andig** [comment]
Ouch, seems they are defined in https://github.com/gonium/gosdm630/blob/master/meters/impl/dzg.go#L24 but not used. Will add asap.

**Comment by frankrichter** [comment]
https://github.com/gonium/gosdm630/blob/master/meters/impl/dzg.go#L25 isn't
correct: 0x000 is Import (active) power and 0x0002 is Export (active) power.
All DZG meter reading are unsigned integers (uint32). To get a signed power
reading you have to calculate Import - Export.

>

**Comment by andig** [comment]
> 0x000 is Import (active) power and 0x0002 is Export (active) power.

Right. This was never tested before, implementation was done blindly.

@ionfan would it be possible to get remote access to your meter, e.g. using ssh or vpn, for testing? Please pm me at cpuidle@gmx.de.
